### ISSUE TITLE
Create features page

### ISSUE THREAD

**Issue by AikedeJongste** [mothstudios/workflow#16]
Create features page and add it to the menu for visitors that are not logged in.

![features](https://cloud.githubusercontent.com/assets/878230/7229678/caf9c866-e766-11e4-906b-e05d4725e6d2.jpg)

**Comment by mothstudios** [comment]
Correct. This is just an example for reference. Still need to work on content for this.
